app/views/images/index.html.erb

```diff
@@ -1,13 +1,13 @@
-<%= stylesheet_link_tag "image" %>
-
-<h1>Welcome!</h1>
-
-<%= link_to 'New image', new_image_path %>
-
-<table class="image_table">
-  <% @images.each do |image| %>
-      <tr>
-        <td><%= image_tag image.imageurl, class: "index_image" %></td>
-      </tr>
-  <% end %>
-</table>
+<div class="album">
+  <div class="container-fluid">
+    <div class="row align-items-center">
+      <% @images.each do |image| %>
+        <a href=<%= "#{images_path}/#{image.id}" %>>
```

Comment:
Refactor this line to put the quotes outside of the ruby expression as such:

```<a href="<%= #{images_path}/#{image.id} %>">```

Another optimization you can do is to just reference the image itself & ruby will be smart enough to take you to the correct images_path route for that image

```<a href="<%= image %>">```

Another optional optimization you can do is use the rails link_to tag http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to

Which would make your html look like the following:

```
 <%= link_to image do %>
        <%= image_tag image.imageurl, class: "card-img mx-auto my-auto" %>
  <% end %>
```

Reply:
RE:  "just reference the image itself & ruby will be smart enough..."
That's really good to know!

I originally tried the `link_to` followed the `image_tag`, but what I really wanted was the whole card to be clickable, not just the image, so I put the link outside of the card.  Regardless, I believe your point was to use the `link_to` instead of the `<a>...</a>` tags?